apps/web/src/locales/en/messages.json

```
@@ -2596,6 +2596,15 @@
       }
     }
   },
+  "viewedCardNumberItemId": {
+    "message": "Viewed Card Number for item $ID$.",
+    "placeholders": {
+      "id": {
+        "content": "$1",
+        "example": "Google"
```

This example is not accurate, it shows the first few characters of the GUID.

```suggestion
        "example": "Unique ID"
```